tests/detect-flowbits/test.rules

```diff
@@ -0,0 +1,13 @@
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb1; sid:1;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb2; sid:2;)
+alert ip any any -> any any (msg:"Flowbit isset ored flowbits"; flowbits:isset,fb3|fb4; sid:3;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb5; sid:5;)
+alert ip any any -> any any (msg:"Flowbit isset ored flowbits"; flowbits:isset,fb5|fb6; sid:6;)
+alert ip any any -> any any (msg:"Flowbit isnotset ored flowbits"; flowbits:isnotset, fb1 | fb2 ; sid:7;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,myflow; sid:8;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,myflow2; sid:9;)
```

Comment:
These rules are also not adding anything new..

Reply:
You would unset `fb1` here and then toggle it later on to make sure `sid:14` is still valid and toggle also works as expected..

Reply:
I see, thanks! In that case, we could combine this one and the toggle, which is also missing a better way of testing, right now, I think.

Reply:
Yas! :D